### ISSUE TITLE
output layer

### ISSUE THREAD

**Issue by AbdulRahmanTaleb** [UPMC-informatique/DeepL_Homology#3]
Bonjour,
Hier on avait dit qu'on allait faire l'output layer de 12 neurones seulement, chacun représentant un bit. MAis après y avoir pensé, est-ce que ça marche vraiment comme ça? puisque l'output layer est censé avoir les logits donc le score pour chaque classe, et puisqu'on a 3901 familles, donc ON DOIT avoir 3901 neurones à la fin non? Si on a que 12 on aura des neurones qui représentent des bits et pas vraiment des classes (pour une meme input, plusieurs neurones doivent etre à 1 pour représenter son label dans ce cas, ce qui n'est pas vraiment la défintion du logits layer), donc est-ce que ça pourrait vraiment marcher?
